packages/calendar/src/features/dual-text-input/DualTextInput.tsx

```diff
@@ -167,14 +169,14 @@ export const DualTextInput: React.FC<DualTextInputProps> = ({
         variant={variant}
         contentRight={
           <Row alignItems={"center"}>
-            <Indent num={0.5}>
-              <FlatButton
-                leftIcon={stenaCalendar}
-                onClick={onClickCalendar}
+            <div style={{ display: "flex" }}>
```

Comment:
Is this div necessary?

Reply:
Nope. Good eye!

Reply:
Using iconRight and OnClickRight since the PR requires the icon to be aligned to the right.